dataland-frontend/src/components/pages/ChooseFrameworkForDataUpload.vue

```diff
@@ -0,0 +1,196 @@
+<template>
+  <AuthenticationWrapper>
+    <TheHeader />
+    <UploaderRoleWrapper>
+      <TheContent>
+        <BackButton id="backButton" label="CHOOSE COMPANY" />
```

Comment:
Label correct?

Reply:
If you come to this page here via the "regular" process (from the choose company page), then this label makes sense. But if you come from somewhere else, this label makes no sense. So just "BACK" might be better, right? What do you think?

Reply:
I would vote to go with "Back" because it is a backButton and if how you mentioned you reach the page from somewhere else then this button won't take you to the choose company page. So i think "BACK" would be better.

Reply:
Vote for "Back"

Reply:
Changed this label to "BACK"

Reply:
Ok